packages/search/src/search.tsx

```diff
@@ -26,13 +42,13 @@ const SEARCH_DEBOUNCE_MS = 300;
 // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Do folks find the `no-empty-function` rule valuable? I see it disabled often, and it's active only on TypeScript files, not on JavaScript ones. Although it checks for things that are not TypeScript specific at all. I wouldn't mind removing it in another PR.

Now when we removed `_.noop` and declare a local `noop = () => {}` per file, there will be many more places that trigger it.

Reply:
I think there's too many places where we declare empty functions to just default to empty behavior. However, ideally, we should check for existence before executing a function, and that way we should be able to avoid most of the cases when we actually need `noop`. While I don't find the rule particularly helpful, I do see it as an opportunity for devs to think about how they can improve their code to not have to rely on an empty function being declared.